compute/cloud-client/src/main/java/compute/snapshotschedule/EditSnapshotSchedule.java

```diff
@@ -17,11 +17,14 @@
 package compute.snapshotschedule;
 
 // [START compute_snapshot_schedule_edit]
+import static com.google.cloud.compute.v1.ResourcePolicySnapshotSchedulePolicyRetentionPolicy.OnSourceDiskDelete.APPLY_RETENTION_POLICY;
```

Comment:
Enums should be prefixed with their class name. Modify the import statement to:

`import com.google.cloud.compute.v1.ResourcePolicySnapshotSchedulePolicyRetentionPolicy. OnSourceDiskDelete;`

Reply:
Thank you! Fixed